src/main/java/io/candydoc/ddd/extract_domain_vocabulary/ReflectionsDomainVocabularyFinder.java

```diff
@@ -161,11 +161,24 @@ private static Set<FullName> dependenciesOf(Class<?> domainModel, Set<Class<?>>
     return dependenciesOfTacticalPattern(domainModel);
   }
 
+  private static void addSuperClassesDeclaredFields(
```

Comment:
Peut-être qu'on pourrait la nommer `addSuperClassesDeclaredFieldsIfAny` ou quelque chose du genre pour dire qu'on ajoute les attributs de la classe parente que si il y en a une.

Reply:
That sounds great. It looks like a good naming practice I didn't know yet.